tests/integration/common-operator-integ-suite.sh

```diff
@@ -224,12 +224,9 @@ main_test() {
     echo "Undeploy Istio"
     ${COMMAND} delete -f "${ISTIO_MANIFEST}" -n "${CONTROL_PLANE_NS}"
 
-    echo "Check that Istio has been deleted"
-    if ${COMMAND} get deployment "istiod" -n "${CONTROL_PLANE_NS}" &>/dev/null; then
-      logFailure "Expected istiod deployment to have been deleted, but it still exists:"
-      ${COMMAND} -n "${CONTROL_PLANE_NS}" get deploy
-      exit 1
-    fi
+    echo "Check that istiod deployment has been deleted (waiting $TIMEOUT)"
```

Comment:
It would be nice to have this test enhanced to show the istiorevision functionality. It could start very simple, such as, creating a istio should also create a istiorevision with same spec.